lazy_tensor_core/OP_LOWERING_GUIDE.md

```diff
@@ -6,7 +6,7 @@ PyTorch wraps the C++ ATen tensor library that offers a wide range of operations
 "Lowering" defines the process of converting lazy tensors computation graphs, which are DAGs (directed acyclic graphs) of ATen operations as captured by executing a model. Plugins, which include the bundled TorchScript plugin, have their own lowering component to native code, but that’s beyond the scope of this documentation. Operations which don't have a lowering will automatically execute using their implementations in the PyTorch core, on CPU and GPU. However, doing so too often defeats the purpose of graph capture through lazy execution, since each such operation forces the evaluation of its inputs, thus fragmenting the captured graph. To maximize the potential of this approach, all operations used by a model should be lowered.
 
 ## Before you start
-You should follow the [instructions](https://github.com/pytorch/pytorch/blob/lazy_tensor_staging/lazy_tensor_core/QUICKSTART.md) to build the lazy tensors component from source. Do this **after** building PyTorch from source using the [standard instructions](https://github.com/pytorch/pytorch/#from-source).
```

Comment:
Why remove this phrase?

Reply:
Because the QUICKSTART.md includes this particular step as well.